### DISCUSSION TITLE
SiteCollectionAdmins - SetPnPTenantSite

### DISCUSSION THREAD

**Discussion by ttd-kevinclement** [pnp/powershell#4012] [Q&A]
Hoping someone can help answer this. Why are some users not valid as site collection admins when using the following commands:

- Add-PnPSiteCollectionAdmin -Owners "user"
- Set-PnPTenantSite -Owners "user"
Errors for the above commands look like such
<img width="1162" alt="Screenshot 2024-06-12 at 3 24 30 PM" src="https://github.com/pnp/powershell/assets/172457461/ad8b6e22-8e29-4a7d-b610-8b5a46978e93">

It seems this behavior is also happening in the OneDrive web gui. What is the criteria for a valid site collection admin?

**Comment by AndersRask** [comment]
Works in my testing, so try and look up the users in Entra ID and see if the account is enabled.
Also policies could restrict adding external users.
